react/IntentOpener/IntentIframe.jsx

```diff
@@ -0,0 +1,41 @@
+/* global cozy */
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class IntentIframe extends React.Component {
+  componentDidMount () {
+    const { action, doctype, options, onComplete } = this.props
+
+    const create = this.props.create || cozy.client.intents.create
+
+    create(action, doctype, {
+        exposeIntentFrameRemoval: true,
+        ...options
+      })
+      .start(this.intentViewer)
```

Comment:
In fact you can add a callback here to be run on created (https://github.com/cozy/cozy-client-js/blob/master/docs/intents-api.md#cozyclientintentscreate):
```
.start(this.intentViewer, onReadyCallback)
```